README.md

```diff
@@ -195,6 +198,68 @@ We have a small collection of algorithms, one for every day of the month. Scroll
   ```
 - **Resources**
   -[Matrix: Determinant](https://www.mathsisfun.com/algebra/matrix-determinant.html)
+
+### **December 11 - Printing Matrix in Spiral Form**
+- **Problem**
+  - Given a Matrix of size `M * N`, output its elements in spiral form.
+  - To better understand the problem statement, consider the following matrix
+  ```
+  01     02     03
+  04     05     06
+  07     08     09
+   ```
+  - The Spiral form of a matrix is as below.
+    ```
+    01 --> 02 --> 03 --> 06 --> 09 --> 08 --> 07 --> 04 --> 05
+    ```
+  - Move Right, Move Down, Move Left, Move Up till you cover all elements of matrix.
+- **Uses**
+- Matrices are used much more in daily life than people would have thought or noticed.
+- Matrices find numerous applications in scientific fields and apply to practical real-life problems as well, thus making an indispensable concept for solving many practical problems.
+- [Applications of matrices in real life problems](https://www.ukessays.com/essays/mathematics/application-of-matrices-in-real-life-problems.php)
+- **Resources**
+  - Matrices are represented using two dimensional arrays in any . So to access a single element in the matrix two “for loops”
```

Comment:
"Matrices are represented using n-dimensional arrays where n is the order of the matrix. In this case have a 2-D matrix, so you will need to use a 2-D array. You need to use two variables to access an element of the matrix. The first variable is used to denote row while the second denotes the column. To traverse the entire matrix, you will need to use two loops.